BuiltInCommands.cpp

```diff
@@ -228,4 +228,25 @@ void KillCommand::execute()
     }
 }
 
-#pragma endregion KillCommand
\ No newline at end of file
+#pragma endregion KillCommand
+
+#pragma region ChangeFileModeCommand
+void  ChangeFileModeCommand::execute()
+{
+    if (cmd_v.size() != 3)
+    {
+       perror("smash error: chmod: invalid arguments");

```

Comment:
Should use the Command::printError function instead, perror is when a syscall fails and errno is populated

Reply:
done